src/lib/common/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function isStringTruthy(str: string): boolean {
+  if (str == null) {
+    return false;
+  }
```

Comment:
To test the _truthiness_ of the `payload` argument, let's allow JavaScript to help decide for us. JavaScript's [rules on _truthiness_](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) do most of what we require, most importantly `null`, `undefined`, and `''` (empty strings) are considered [_falsy_](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).
```suggestion
  if (!payload) return false;
```

**Note**: The `!` (negation) operator is used to determine the _falsiness_ of the `payload` argument.

Reply:
Great suggestion!